### PR TITLE
Added insertion_sort | Closes #1

### LINKED ISSUE #1: Add insertion sort
Add insertion sort with test cases

### PR DESCRIPTION
**Pull request by ranveeraggarwal**

### CODE REVIEW

File: insertion_sort.py [hunk 1]
```diff
@@ -23,7 +11,12 @@ def insertion_sort(lis):
     if lis == list():
```
Comment (DheerendraRathor) [inline insertion_sort.py]: This is not required. Also add a test case with empty list.
